system/Security/Exceptions/SecurityException.php

```diff
@@ -16,20 +16,20 @@
 
 class SecurityException extends FrameworkException implements HTTPExceptionInterface
 {
-    public static function forDisallowedAction()
+    public static function forDisallowedAction(): SecurityException
```

Comment:
as kenjis mentioned, please revert this and use `@return self` instead

Reply:
Done.